bridge-impl/src/main/java/com/liferay/faces/bridge/component/internal/ResourceComponent.java

```diff
@@ -31,6 +31,10 @@
  */
 public class ResourceComponent extends UIOutput {
 
+	public ResourceComponent() {
+		// Defer initialization of ResourceComponent until restoreState(FacesContext, Object) is called.
```

Comment:
@stiemannkj1 Is this zero-arg constructor here for a similar reason to the one listed here? https://github.com/liferay/liferay-faces-bridge-impl/blob/master/bridge-impl/src/main/java/com/liferay/faces/bridge/renderkit/html_basic/internal/ResourceRendererBridgeImpl.java#L59